main/com.tunein/manifest.tt

```diff
@@ -0,0 +1,111 @@
+// conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above
+//    copyright notice, this list of conditions and the following
+//    disclaimer in the documentation and/or other materials
+//    provided with the distribution.
+// 3. Neither the name of the copyright holder nor the names of its
+//    contributors may be used to endorse or promote products derived
+//    from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+// FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+// COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+// INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+// (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+// OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Author: Jake Wu <jmhw0123@gmail.com>
+
+class @com.tunein
+#_[thingpedia_name="Tunein"]
+#_[thingpedia_description="Listen to internet radio"]
+#_[canonical="radio"]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="media"]
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+  
+  entity station #_[description="Radio stations on TuneIn"];
+  entity show #_[description="Show names from the station on TuneIn"];
+
+  list query station(out id: Entity(com.tunein:station)
+                     #[unique=true]
+                     #[string_values="com.tunein:station"]
+                     #_[canonical={
+                       default="base",
+                       base=["name", "title"],
+                       property=["name #", "title #", "channel #"],
+                       passive_verb=["named #", "called #", "titled #"],
+                     }],
+                     out show: Entity(com.tunein:show)
+                     #[unique=true]
+                     #[string_values="com.tunein:show"]
+                     #_[canonical={
+                       default="base",
+                       base=["name", "title", "track", "show", "program", "current show", 
+                             "current program", "now playing", "currently playing"],
+                       passive_verb=["named #", "called #", "titled #", "related to #"],
+                       property=["name #", "title #", "now playing #", "currently playing #"]
```

Comment:
"now playing #" is a passive_verb
(technically, a gerund grammatically but it goes in the same category of canonical)

`property` canonical form "with <param> <value>", so you would end up with "with now playing <value>" which is not correct.